.github/workflows/tests.yml

```
@@ -56,6 +56,16 @@ jobs:
           unzip -o dcm2niix_lnx.zip
           sudo install dcm2nii* /usr/bin/
 
+      - name: SpinalCord Toolbox installation
+        run: |
+          bash
+          cd ../..
+          git clone --depth 1 https://github.com/neuropoly/spinalcordtoolbox
+          cd spinalcordtoolbox
+          yes | ASK_REPORT_QUESTION=false PIP_PROGRESS_BAR=off ./install_sct
```

Could you try [cloning the branch](https://stackoverflow.com/questions/1911109/how-do-i-clone-a-specific-git-branch) from https://github.com/neuropoly/spinalcordtoolbox/pull/3102 temporarily to test how that PR affects this issue?

```suggestion
          git clone --depth 1 --single-branch --branch ng/strict-shell https://github.com/neuropoly/spinalcordtoolbox
          cd spinalcordtoolbox
          ./install_sct -y
```

---

So far so good on macOS 10.15 🎉

---

And we have a winner on macOS 11.0!

---

Just to fill in the context, @kousu identified the problem here: https://github.com/actions/runner/issues/884

`yes |` was indeed the issue, as you had identified. :sweat_smile:

---

Well I don't know if I'd say I *identified* it, but it's narrowed down. I'm pretty sure it's Github's bug. But they missed it because they don't expect people to be doing something fragile like `yes |` in this day and age.

---

But the same bug should show up if someone uses `seq | while read; do ... if [something]; then break; fi; done`?

---

Yeah, it's not specifically `yes`, rather the piping, correct? Because the same hang still happened when I tried piping the exact number of `'y'`s from `printf()`.